activation/atxdb.go

```diff
@@ -636,6 +636,11 @@ func (db *DB) GetEpochWeight(epochID types.EpochID) (uint64, []types.ATXID, erro
 		}
 		weight += atxHeader.GetWeight()
 	}
+
+	if weight == 0 {
+		weight = 1
+	}
```

Comment:
I don't like this change. I assume it's there to prevent division by zero, however it should not be done here, but at the division site. You actually have a check for this condition here:

https://github.com/spacemeshos/go-spacemesh/blob/b341df7b58bc98f0d86b680de0a4c286f0500234/tortoisebeacon/tortoise_beacon.go#L814-L816

Due to this change, the check above (that would actually print an error message) can never catch this condition.